certified-connectors/Office365GroupsMail/apiDefinition.swagger.json

```diff
@@ -0,0 +1,768 @@
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "$ref": "#/definitions/Post"
+                        }
+                    }
+                }
+            }
+        },
+        "/v1.0/groups/{groupId}/threads/{threadId}/posts/{postId}/attachments": {
+            "get": {
+                "operationId": "GetAttachments",
+                "description": "Retrieve a list of attachments that belong to a post.",
+                "summary": "List the attachments of a post",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/groupId__in_path"
+                    },
+                    {
+                        "name": "threadId",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "x-ms-url-encoding": "single",
+                        "description": "Enter the conversation thread ID.",
+                        "x-ms-summary": "Thread ID"
+                    },
+                    {
+                        "name": "postId",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "x-ms-url-encoding": "single",
+                        "description": "Enter the post ID.",
+                        "x-ms-summary": "Post ID"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "@odata.context": {
+                                    "type": "string",
+                                    "description": "The @odata.context link.",
+                                    "x-ms-visibility": "internal"
+                                },
+                                "value": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/Attachment"
+                                    },
+                                    "description": "Array containing the attachments of the post."
+                                }
+                            }
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "@odata.nextLink"
+                }
+            }
+        },
+        "/v1.0/me/memberOf/$/microsoft.graph.group": {
+            "get": {
+                "operationId": "ListGroups",
+                "description": "Retrieves all Office 365 Groups the user is part of.",
```

Comment:
Should it be Microsoft instead of Office?

Reply:
True! I'll update all the occurrences.